src/collection-view.js

```diff
@@ -499,6 +506,30 @@ Marionette.CollectionView = Marionette.View.extend({
     return childViews;
   },
 
+  // Return true if the given child should be shown
+  // Return false otherwise
+  // The filter will be passed (child, index, collection)
+  // Where
+  //  'child' is the given model
+  //  'index' is the index of that model in the collection
+  //  'collection' is the collection referenced by this CollectionView
+  testChild: function (child, index) {
```

Comment:
I've always thought any use of "child" is confusing. "Is it an `ItemView` or a `Model`?"